qfile.c

```diff
@@ -57,11 +60,77 @@ struct qfile_opt_state {
 	bool slotted;
 	bool basename;
 	bool exact;
+	bool skip_plibreg;
 	bool orphans;
 	bool assume_root_prefix;
 };
 
 /*
+ * As a final step, check if file is in the plib_reg
+ */
+static int qfile_check_plibreg(void *priv)
+{
+	struct qfile_opt_state *state = priv;
+
+	int fd_plibreg;
+        FILE *fp_plibreg;
+	struct stat cst;
+
+	/* Open plibreg */
+        fp_plibreg = NULL;
+	fd_plibreg = open("/var/lib/portage/preserved_libs_registry", O_RDONLY|O_CLOEXEC, 0);
```

Comment:
this needs EPREFIX prepended here one way or the other, I suspect there's actually some defines that point to locations like these, we should use one

Reply:
Fixed

Reply:
Ah, the change uses eprefix which is defined in main.c. This worked on v0.74 but not here on git master. Something changed in the relationship between the applet and main.c. Will look into it. Happy to receive pointers if you have any.

Reply:
qmerge.c:               return qmerge_add_set_file(CONFIG_EPREFIX, "/var/lib/portage",

wouldn't that work?

Reply:
Yep, that works.